### PR TITLE
!feat: update project story fields

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1621

Renames `regen:landStewardStory` and `regen:landStewardStoryTitle` to `regen:story` and `regen:storyTitle`

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: ops/C01/project-metadata/C01-002-project-page.jsonld [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "https://regen-registry.s3.amazonaws.com/projects/kasigau/image4.png"
   ],
   "regen:landStewardStory": "In 1998, the local community supported our plan to establish the Rukinga Wildlife Sanctuary that covers 80,000 acres of forest. We established a community works project so local residents had an alternative income stream in place of poaching and clear cutting​. We brought on locally hired rangers and trained them to be wilderness guardians. We convinced the owners of the cattle to remove the cattle from the land to reduce conflict over resources.",
-  "regen:landStewardStoryTitle": "The project area is home to a fantastic diversity of over 50 species of large mammals including elephants, zebras, lions, cheetahs and leopards.",
+  "regen:landStewardStoryTitle": "The project is home to over 50 species of large mammals including elephants, zebras, lions, cheetahs and leopards.",
```
Comment (blushi) [inline ops/C01/project-metadata/C01-002-project-page.jsonld:28]: see this thread for explanation: https://www.figma.com/file/TyhFaXqko9F8OvLEE4Zomg?node-id=2:4283#411798832

File: shacl/projects/common.ttl [hunk 2]
```diff
@@ -16,7 +16,7 @@ regen:NamePropertyShape sh:path schema:name ;
 .
 
 regen:DescriptionPropertyShape sh:path schema:description ;
-  sh:maxLength 1400 ;
+  sh:maxLength 600 ;
```
Comment (blushi) [inline shacl/projects/common.ttl:19]: see https://www.figma.com/file/TyhFaXqko9F8OvLEE4Zomg?node-id=2:4283#411792576

**Comment by blushi** [comment]
Needs https://regen-network.slack.com/archives/CBEMQ4QDD/p1681216011319909 for the SHACL validation to pass since we lowered the project description max length to 600 char

**Comment by blushi** [comment]
@wgwz ready for review now

**Review by wgwz — APPROVED** [review]
LGTM

**Pull request merged**